ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/errdisable.j2

```diff
@@ -59,7 +59,7 @@ errdisable recovery cause {{ cause }}
 {%                 endif %}
 {%             endfor %}
 {%         endif %}
-{%         if errdisable.recovery.interval is defined and errdisable.recovery.interval is not none  %}
```

Comment:
missing check for `errdisable.recovery`. Add it above the `causes` section and nest both `causes` and `interval` under that `if`